From 49ea4351cecb53b42eb788212c764c3c8b934015 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Thu, 14 Nov 2024 14:42:27 +0100
Subject: [PATCH 1400/1695] dts: stm32: remove provider-cells properties for
 stm32mp21x

Removes theses properties:
- #clock-cells
- #reset-cells

from scmi node, as theses nodes are not resources provider as same as
their equivalent on the Linux side.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git a5ed7562aa6d5db8a6f0c7918eb368c669d4604c dts: stm32: remove provider-cells properties for stm32mp21x]
Fixes: ce64cb4654 ("dts: stm32: describe scmi resources for stm32mp21x")
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: I6f7f70f8e0c604f11b2102c719f63af7a4bbceef
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/418590
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/dts/stm32mp211.dtsi | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp211.dtsi b/core/arch/arm/dts/stm32mp211.dtsi
index 6a459179b..4b4dd5629 100644
--- a/core/arch/arm/dts/stm32mp211.dtsi
+++ b/core/arch/arm/dts/stm32mp211.dtsi
@@ -153,7 +153,6 @@
 				clocks {
 					#address-cells = <1>;
 					#size-cells = <0>;
-					#clock-cells = <1>;
 
 					clock@0 {
 						reg = <CK_SCMI_ICN_HS_MCU>;
@@ -975,7 +974,6 @@
 				resets {
 					#address-cells = <1>;
 					#size-cells = <0>;
-					#reset-cells = <1>;
 
 					reset@0 {
 						reg = <RST_SCMI_C1_R>;
@@ -1067,7 +1065,6 @@
 				clocks {
 					#address-cells = <1>;
 					#size-cells = <0>;
-					#clock-cells = <1>;
 
 					clock@0 {
 						reg = <CK_SCMI_ICN_HS_MCU>;
@@ -1889,7 +1886,6 @@
 				resets {
 					#address-cells = <1>;
 					#size-cells = <0>;
-					#reset-cells = <1>;
 
 					reset@0 {
 						reg = <RST_SCMI_C1_R>;
-- 
2.47.2

